src/cli/src/commands/create-build.ts

```diff
@@ -70,12 +73,26 @@ export const handler = async (args: Args): Promise<{}> => {
   const { timestamp, name, subject } = await Git.getRevisionDetails(revision, config.cwd);
   const branch = await Git.getBranch(config.cwd);
 
+  const getRevisionItem = (): BuildMetaItem => {
```

Comment:
Really minor stylistic change: it doesn't seem necessary to construct a function here.

```js
let revision = await Git.getCurrentRevision(config.cwd);
if (config.buildUrlFormat) {
  const toPath = pathToRegexp.compile(config.buildUrlFormat);
  const url = toPath({ revision });
  revision = {
    value: revision,
    url
  };
} 
```